plugins/janus_videoroom.c

```diff
@@ -2427,11 +2471,11 @@ static void *janus_videoroom_handler(void *data) {
 				publisher->vrc = NULL;
 				janus_mutex_init(&publisher->rec_mutex);
 				publisher->firefox = FALSE;
-				publisher->bitrate = videoroom->bitrate;
+				publisher->bitrate = publisher->room->bitrate;
```

Comment:
Why changing this? Isn't it more likely that `publisher->room` is broken (user left) than `videoroom` is (room destroyed)?